### ISSUE TITLE
future annual assessments

### ISSUE THREAD

**Issue by emseabee** [tbep-tech/tbnmc-compliance-assessment-2022#6]
need to be parsed by LTB remainder, like the 2022 RA

**Comment by fawda123** [comment]
Yep, also talked to @esherwoo77 about this.  It's doable but will likely require some manual data pulls since Manatee and Pinellas data aren't as easy to get as Hillsborough.
